src/pe_reports/data/data_schema.sql

```diff
@@ -60,7 +60,7 @@ CREATE TABLE public.executives
 CREATE TABLE IF NOT EXISTS public."DNSTwist"
 (
     dnstwist_uid uuid default uuid_generate_v1() NOT NULL,
-    "discoveredBy" text NOT NULL,
+    "discoveredBy" uuid NOT NULL,
```

Comment:
```suggestion
    "discoveredBy" text NOT NULL,
```
There is already a UUID column. Is it the intention to have a second UUID?

Reply:
This resolves the Postgresql error concerning keys with different data types.  The UUID column referenced here is a foreign key with it's parent being the domains table column domain_uid.

Reply:
Ok that makes sense. I understand the relationship.